### PR TITLE
Refactors and Things

### PR DESCRIPTION
**Pull request by iREDMe**
* Makes the ``for (let i = 0; i <...`` to ``for (const item of body.body)``

* Made some strings with ``'str' + 'str'`` to template literals (yes I did google them don't hurt me)

* Also for some items I have no idea if they work. I do know that fetching Bot, Server, and User does work, though Upvotes idk :P

### CODE REVIEW

File: lib/Client.js [hunk 1]
```diff
@@ -17,10 +17,10 @@ const Server = require('./Structures/Server');
  */
 class Client {
 	/**
-	 * @param {String} id The ID of the bot.
-	 * @param {String} token The token provided from the bot's token page.
+	 * @param {String} [id=''] The ID of the bot.
```
Comment (PassTheMayo) [inline lib/Client.js]: This makes Jsdoc think that `id` is optional

File: lib/Client.js [hunk 2]
```diff
@@ -94,43 +95,22 @@ class Client {
 	 * Retrieves information about a specific bot.
 	 * @memberof Client
 	 * @returns {Promise<Collection|HTTPError>}
-	 * @param {Boolean} idsOnly Use true if you only want to return IDs of users who upvoted in the last 24 hours. False for all user data.
+	 * @param {Boolean} [idsOnly=false] Use true if you only want to return IDs of users who upvoted in the last 24 hours. False for all user data.
```
Comment (PassTheMayo) [inline lib/Client.js]: Also makes it optional

**Comment by PassTheMayo** [comment]
I'll finish reviewing this later, I'm a bit busy :^)

File: lib/Client.js (hunk 2 above)
Comment (iREDMe) [reply lib/Client.js]: I made it optional because I thought making it default to `false` would make more sense

**Comment by iREDMe** [comment]
Made ID as a required param now

**Comment by iREDMe** [comment]
(also made more template literal things)

**Comment by iREDMe** [comment]
Refactoring

**Comment by PassTheMayo** [comment]
Although this PR could really ~~useless tbh~~ could improve our coding style :^), we like it the way it is. Sorry about that but I'm gonna decline this one. And our single quotes with + variables is just a preference, and I cba to adapt to it :notLikeThis:

**Comment by iREDMe** [comment]
`could really improve*` not `could really could`

**Pull request closed without merging**